Infrastructure/How to Install and Use Geospatial R Packages.md

```diff
@@ -91,6 +91,9 @@ Many of the dependencies of the geospatial R packages can be installed as binari
 # Get list of geospatial package dependencies that can be installed as binaries
 geo_deps_bin <- sort(setdiff(geo_deps, geo_pkgs))
 
+# Remove available (base) packages
+geo_deps_bin <- geo_deps_bin[!(geo_deps_bin %in% rownames(available.packages()))]
+
```

Comment:
```suggestion
# Remove packages that are already installed from the list of geospatial package dependencies
geo_deps_bin <- sort(setdiff(geo_deps_bin, as.data.frame(installed.packages())$Package))

```

My suggestion above removes packages that are already installed from the list of geospatial package dependencies, which runs faster, and should prevent any existing packages from being upgraded unintentionally.